src/app/list/state/list/list.effects.ts

```diff
@@ -0,0 +1,22 @@
+import {Actions, createEffect, ofType} from "@ngrx/effects";
+import {Injectable} from "@angular/core";
+import * as TaskActions from "./list.actions"
+import {tap} from "rxjs/operators";
+import {selectTasks} from "./list.selectors";
+import {Store} from "@ngrx/store";
+import {Task} from "./list.state";
+
+@Injectable()
+export class ListEffects {
+  // we need this effect to store actions changes in localStorage
+  saveToLocalStorage$ = createEffect(() => this.actions$.pipe(
```

Comment:
typically this is what localStorageSync does if you set it up correctly

Reply:
ah ok, I will change it